op-node/testutils/mock_l1.go

```diff
@@ -12,6 +13,7 @@ type MockL1Source struct {
 }
 
 func (m *MockL1Source) L1BlockRefByLabel(ctx context.Context, label eth.BlockLabel) (eth.L1BlockRef, error) {
+	fmt.Println("L1BlockRefByLabel", label)
```

Comment:
Is this intentional?

Reply:
it was quite useful for debugging, but I should probably put it behind a logger. Will drop it from this PR.